### PR TITLE
Add basic routing to home and sign in/up

### PR DESCRIPTION
**Pull request by m0nd**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
+import Header from "./components/Header/Header";
 import Home from "./pages/Home";
 
 function App() {
   return (
-    <div>
-      <Home />
-    </div>
+      <div>
+        {/* <Home /> */}
```
Comment (sekayasin) [inline src/App.js]: we can remove this comment

**Review by sekayasin — CHANGES_REQUESTED** [review]
from the home Page -- we can also remove/comment the Footer component since we have it now in the main App component.

File: src/App.js (hunk 1 above)
Comment (m0nd) [reply src/App.js]: Ok will do that

**Review by sekayasin — APPROVED** [review]
LGTM

**Pull request merged**